bitbots_dynup/scripts/generate_walkready_animation.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+import sys
+
+import actionlib
+import rospy
+from actionlib_msgs.msg import GoalStatus
+from geometry_msgs.msg import Twist
+from bitbots_msgs.msg import JointCommand, DynUpAction, DynUpGoal
+import math
+
+
+def callback(msg):
+    output_string = "{\n \"author\": \"walkready_script\", \n \"description\": \"none\",\n \"hostname\": \"tams159\", \n \"keyframes\": [ \n { \n \"duration\": 1.0, \n \"goals\": { \n"
```

Comment:
I don't think this animation is used anywhere in the dynup. Unless @SammyRamone disagrees, I'll remove it.

Reply:
The walkready animation is still used in the HCM to check if the robot is currently in the walkready pose (e.g. during startup). So don't delete it.
It would be greate to share parameters between walking and dynup, but it think this is very difficult, we discussed this some time back.